pygerduty/__init__.py

```diff
@@ -460,6 +460,7 @@ class Service(Container):
     def __init__(self, *args, **kwargs):
         Container.__init__(self, *args, **kwargs)
         self.email_filters = EmailFilters(self.pagerduty, self)
+        self.escalation_policy = EscalationPolicies(self.pagerduty, self)
```

Comment:
You can only have 1 escalation policy for a service so being plural would in my opinion be wrong